db/migrate/20220428193223_create_pools.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePools < ActiveRecord::Migration[6.1]
```

Comment:
You created a migration but forgot to run `rails db:migrate` and update the schema!